src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.scss

```diff
@@ -32,6 +32,13 @@ cds-header-item {
   margin-left: 6rem;
 }
 
+::ng-deep .cds--header__menu-title[aria-expanded="true"] + .cds--header__menu {
```

Comment:
why not add a class in html code and have this selector simplified?

Reply:
@cloudbehl , I did try that as well but it didn't work that way. Maybe am missing something related to carbon styling methods but could make it work using these selectors only.

Reply:
@nizamial09 @ivoalmeida i'm not sure, can you help @aaSharma14